### ISSUE TITLE
lynx: SSL error: Can't find common name in certificate

### ISSUE THREAD

**Issue by rashil2000** [ScoopInstaller/Main#2855]
## Bug Report

**Package Name:** lynx

### Current Behaviour

Websites using HTTPS do not open at all.

### Expected Behaviour

All websites should open.

### Additional context/output

The dependency `cacert` might be the cause of the problem.

```
❯ lynx

Looking up lynx.invisible-island.net
Making HTTPS connection to lynx.invisible-island.net

lynx: Can't access startfile https://lynx.invisible-island.net/
```

### System details

**Windows version:** 10.0.19043.1288
**OS arch (32 or 64 bit):** 64
**PowerShell version:** 7.1.5
**Additional software:** none

**Comment by rashil2000** [comment]
It has been like that for quite some time - https://github.com/ScoopInstaller/Main/pull/512#pullrequestreview-308197679 - I'm not sure it will work in stable either.

The actual problem might be this - mentioned on the Lynx homepage
![image](https://user-images.githubusercontent.com/46838874/142023292-ad3a3f03-5636-493b-a55b-b047274bc7dd.png)

The manifest currently downloads OpenSSL 1.1.1, but that is not supported it seems. And OpenSSL 1.1.0 is hard to find anywhere.

**Comment by tech189** [comment]
You're right. Just tested the stable release with OpenSSL 1.1.1l and it doesn't work. Looks like there are security vulnerabilities with version 1.1.0 so that's why it's hard to find - I don't recommend we go looking for it and introduce vulnerabilities for scoop users! So, it looks like it's up to lynx to add support for 1.1.1 which was released 11 September 2018. Or better yet the latest version 3.0.0 which was recently released (7 September 2021).

**Comment by rashil2000** [comment]
Maybe we can ask the author what's going on - would you be willing to file a report in Lynx? (Ref: https://lists.gnu.org/archive/html/lynx-dev)

**Comment by tech189** [comment]
It looks like someone asked a similar question just over a year ago: https://lists.gnu.org/archive/html/lynx-dev/2020-09/msg00005.html

And this was the author's reply: https://lists.gnu.org/archive/html/lynx-dev/2020-09/msg00006.html

Essentially, it looks like https support for Windows is not planned/being worked on at the moment. A workaround is to install it via WSL or Cygwin instead. ¯\\\_(ツ)_/¯

**Comment by rashil2000** [comment]
I see. However our issue is slightly different - that of difference in Windows OpenSSL versions. But that might very well be due to the author not being aware/having time to look at Windows builds. 😥

**Comment by tech189** [comment]
I can't even get the regular installer to work, it fails to install with the error that it 'Failed to copy msvcr120.dll'. That file is from the Visual C++ Redistributable Package for Visual Studio 2013 but even with both the 32bit and 64bit versions installed it still fails. 😢

**Comment by rashil2000** [comment]
I remember the installer worked when I tried it around a month ago, but yeah now all of them fail with the same error 😕

**Comment by tech189** [comment]
Sad times... I would be grateful if you could make a bug report about the installer on that mailing list, I'm not so comfortable posting my real name and email there!

**Comment by rashil2000** [comment]
I'll see to it if I get time, but I'm not sure.

> I'm not so comfortable posting my real name and email there!

You don't have to use your real name/email 😉. It's just for communication, they're not going to verify you. 😏

**Comment by tech189** [comment]
Of course, but I couldn't see the option to leave my username - the button to reply to a thread opens your mail client to respond by email!
